doc/build.md

```diff
@@ -29,16 +29,15 @@ Before building the project, consult the following table to ensure you use the r
 
 #### 2. Download and Build Pixar USD 
 
-See Pixar's official github page for instructions on how to build USD: https://github.com/PixarAnimationStudios/USD . Pixar has recently removed support for building Maya USD libraries/plug-ins in their github repository and ```build_usd.py```. When building the maya-usd project, it is important the recommended ```Pixar USD``` and ```Animal Logic``` commitID or tag from the table below are used: 
-
-|   Required    |      ![](images/pxr.png)          |         ![](images/al.png)            |
-|:------------: |:---------------:  |:-----------------:    |
-| CommitID/tag  | 40c8134(>20.02)   | 631a2911(>0.34.0)     |
+See Pixar's official github page for instructions on how to build USD: https://github.com/PixarAnimationStudios/USD . Pixar has recently removed support for building Maya USD libraries/plug-ins in their github repository and ```build_usd.py```. When building the maya-usd project, it is important the recommended ```Pixar USD``` commitID or tag from the table below are used: 
 
+|               |      ![](images/pxr.png)          |        
+|:------------: |:---------------:                  |
+|  CommitID/Tags | master: [v19.07](https://github.com/PixarAnimationStudios/USD/tree/v19.07) or [v19.11](https://github.com/PixarAnimationStudios/USD/tree/v19.11) or [v20.02](https://github.com/PixarAnimationStudios/USD/tree/v20.02) <br> dev: [40c813](https://github.com/PixarAnimationStudios/USD/commit/40c813421425c47aa84f4572b4798fbb3430984e) | 
```

Comment:
Hey @HamedSabri-adsk, small suggestion here, but for the tagged releases, maybe linking to the actual release as opposed to the code tree might be better? For example:
[v19.07](https://github.com/PixarAnimationStudios/USD/releases/tag/v19.07)

That takes you to the release page where you can download the source as a zip or tarball if you want to, or you can click through to get to the code tree.

Reply:
@mattyjams  Ugh great point. Sure, fixing it now.

Reply:
Please see 3108d4c

Reply:
Awesome, thanks @HamedSabri-adsk!